charts/guac/values.yaml

```diff
@@ -190,10 +200,26 @@ guac:
     ingest: false
     jobName: ingest-guac-data
 
+  ingress:
+    enabled: false
+    # ingressClassName:
+    # webuiHostname:
+    # apiHostname:
+
+  backend:
+    ent:
+      db-driver: postgres
+      db-address: postgres://guac:guac@host:port/dbName?sslmode=disable
+      db-migrate: true
+      db-debug: false
+
+  pubSubAddr: awssqs://sqs.[aws_region].amazonaws.com/[account_id]/[queue_name]?region=[aws_region]
+  blobAddr: s3://[bucket_name]?region=[aws_region]
```

Comment:
So we could default this to a local PV. Did something similar with Tilt in the guac main repo -https://github.com/guacsec/guac/pull/1666/files#diff-5ae789a43df3c5e816feb734327e53b6f10d07e364088154c8464626c87f5118

Otherwise, this will be broken for users that want to use the helm charts from [guac](https://docs.guac.sh/setup/)